TrackingTools/TrackAssociator/python/DetIdAssociatorESProducer_cff.py

```diff
@@ -33,7 +33,7 @@
     etaBinSize = cms.double(0.125),
     nEta = cms.int32(48),
     nPhi = cms.int32(48),
-    includeBadChambers = cms.bool(False)
+    includeBadChambers = cms.bool(True)
```

Comment:
... maybe we actually add this to Configuration/StandardSequences/python/Reconstruction_Data_cff.py ?
I think it's more appropriate than even doing it just for run2